src/core/live_trackers/cpp/live_deviceio_factory.cpp

```
@@ -16,12 +16,54 @@
+    for (const auto& ext : XrTimeConverter::get_required_extensions())
+        all.insert(ext);
+
+    for (const auto& tracker : trackers)
+    {
+        if (!tracker)
+            throw std::invalid_argument("LiveDeviceIOFactory: null tracker in trackers list");
+
+        try_add_extensions<HeadTracker, LiveHeadTrackerImpl>(*tracker, all) ||
+            try_add_extensions<HandTracker, LiveHandTrackerImpl>(*tracker, all) ||
+            try_add_extensions<ControllerTracker, LiveControllerTrackerImpl>(*tracker, all) ||
+            try_add_extensions<FullBodyTrackerPico, LiveFullBodyTrackerPicoImpl>(*tracker, all) ||
```

Cppcheck (2.20.0)

[style] The function 'update' is never used.

(unusedFunction)